modules/account-lib/src/coin/hbar/transaction.ts

```diff
@@ -75,6 +99,22 @@ export class Transaction extends BaseTransaction {
   body(tx: proto.Transaction) {
     this._txBody = proto.TransactionBody.decode(tx.bodyBytes);
     this._hederaTx = tx;
+    // this.loadPreviousSignatures();
+  }
+
+  /** Decode previous signatures from the inner hedera transaction
+   * and save them into the base transaction signature list.
+   */
```

Comment:
`/** Decode previous signatures from the inner hedera transaction and save them into the base transaction signature list. */`

Or

```
/** 
   * Decode previous signatures from the inner hedera transaction
   * and save them into the base transaction signature list.
   */
```

Choose one or the other but make sure it does not exceed 120 chars per line

Reply:
Will do